winml/lib/Api/LearningModelSession.cpp

```diff
@@ -136,8 +136,15 @@ void LearningModelSession::Initialize() {
 
     allow_spinning = session_options_impl->GetIntraOpThreadSpinning();
     num_intra_op_threads = session_options_impl->GetIntraOpNumThreads();
+
+    const auto& paths = session_options_impl->GetCustomOpLibraryPaths();
+    for (const auto& path : paths)
+    {
```

Comment:
nit: doesn't match bracket style from the rest of this file

Reply:
moved brace to same line